packages/Modal/stories/Modal.stories.js

```diff
@@ -72,6 +72,31 @@ storiesOf("Modal", module)
       <DemoFullWidthContent />
     </ModalStory>
   ))
+  .add("with form render", () => (
+    <Modal
+      isOpen
+      as="form"
+      onSubmit={event => {
+        alert("submit");
+        event.preventDefault();
+      }}
+    >
+      <Modal.Content>
+        <input type="text" defaultValue="your text" />
+      </Modal.Content>
+      <Modal.Footer>
+        <Button
+          isSubmit
+          onClick={() => {
```

Comment:
This is not used I think

Reply:
if the onClick is not included the onSubmit will not work when the button gets clicked, do you prefer to have a void/noop function and remove the console? unless we want to add a new event like onClickSubmit.

Reply:
That's surprising: we have a defaultProp for `onClick` which is `() => {}`, so it should work 🤔

Reply:
your right, my bad, is working.